packages/sources/por-address-list/src/transport/bedrockUniBTC.ts

```diff
@@ -5,13 +5,14 @@ import { BaseEndpointTypes } from '../endpoint/bedrockBTC'
 interface ResponseSchema {
   btc: {
     type: string
+    bridge_source: string
     addr: string
   }[]
   evm: {
     [key: string]: {
       chain_id: number
       vault: string
-      tokens: string[]
+      tokens: string[][]
```

Comment:
Could we leave a comment describing this type? `token[1]` later is a bit obscure as reader doesn't know what is in `token`.

Even better if we can actually add a labeled tuple, e.g.
```
type token = [name: string, id: string, value1: string, value2: string]
```

Reply:
Done